app/forms/idv/api_document_verification_status_form.rb

```diff
@@ -31,12 +31,17 @@ def remaining_attempts
 
     def timeout_error
       return unless @async_state.timed_out?
-      errors.add(:timeout, t('errors.doc_auth.document_verification_timeout'))
+      errors.add(
+        :timeout, t('errors.doc_auth.document_verification_timeout'),
+        type: :document_verification_timeout
+      )
     end
 
     def failed_result
       return if !@async_state.done? || @async_state.result[:success]
-      @async_state.result[:errors].each { |key, error| errors.add(key, error) }
+      @async_state.result[:errors].each do |key, error|
+        errors.add(key, error, type: :error)
```

Comment:
My worry is that the single `type: :error` combines all these different errors into one type, is that ok here?